### PR TITLE
Midterm

### PR DESCRIPTION
**Pull request by ceciliachan1979**

### CODE REVIEW

File: Exams/Rutgers-Analysis/midterm.tex [hunk 1]
```diff
@@ -0,0 +1,142 @@
+\documentclass{article}
+\usepackage{amssymb}
+\usepackage{nth}
+\usepackage[utf8]{inputenc}
+\usepackage{graphicx}
+\usepackage[final]{pdfpages}
+
+
+\title{Number Theory}
```
[the remark this answers is not in the document]
Comment (ceciliachan1979) [reply Exams/Rutgers-Analysis/midterm.tex:9]: 明顯 copy and paste error 😝

**Comment by E7-87-83** [comment]
Problem 2 嘅 presentation 可以做好啲
例如，我未見過 math book 咁寫
image
equal sign 可以「 indent」, 但greater than 、 less than 通常每行都寫曬兩邊

**Comment by E7-87-83** [comment]
去證一個負數 x 嘅absolute value 少於 A，通常咁做

Prove  x < 0
Prove -x < A
Conclude |x| <A

去證一個正數 x 嘅absolute value 少於 A，通常咁做

Prove  x > 0
Prove  x < A
Conclude |x| <A

負數嗰部分，我地唔會喺line 2畫蛇添足寫成 -x < 0 < A

**Comment by E7-87-83** [comment]
Prob 1

Prob 1.3: 問嘅係 p => q 係 true 定 false 。簡單答 false 即可。

Other parts are correct.

**Comment by E7-87-83** [comment]
Prob 3

Presentation 需要改善。
line 1: A < B < C
line 2: D < B < E

咁代表乜？ "A=D and C=E" 、" D<A and C<E" 、"A=D and C<E" 、"D<A and C=E" (仲有greater-than-or-equal-to-sign嘅cases，略)定係 line 1 跟 line 2 無因果關係？？

**Comment by E7-87-83** [comment]
https://en.wikipedia.org/wiki/Language_of_mathematics

**Comment by ceciliachan1979** [comment]
> Prob 3
> 
> Presentation 需要改善。 line 1: A < B < C line 2: D < B < E
> 
> 咁代表乜？ "A=D and C=E" 、" D<A and C<E" 、"A=D and C<E" 、"D<A and C=E" (仲有greater-than-or-equal-to-sign嘅cases，略)定係 line 1 跟 line 2 無因果關係？？

line 1 去 line 2 只係全部 + 2

**Comment by E7-87-83** [comment]
Prob 4 OK, 但似乎頭昏用錯字，已經證明咗limit存在，但之後"_Suppose_ the limit exists,"

另外，我會寫多一啲details

> 
> Since the limit exists, let the limit be $x_0$, then 
> 
> \begin{eqnarray*}
>    x_0
>   & = & \lim_{n \to \infty} a_n \\
>   &=& \lim_{n \to \infty} \sqrt{4 + a_{n-1}} \\
>   &=& \sqrt{4 + \lim_{n \to \infty} a_{n-1}} \\
>   &=& \sqrt{4 + x_0 } \\
> \end{eqnarray*}
>

**Comment by E7-87-83** [comment]
Prob 6 - OK

**Comment by ceciliachan1979** [comment]
> 去證一個負數 x 嘅absolute value 少於 A，通常咁做
> 
> Prove x < 0 Prove -x < A Conclude |x| <A
> 
> 去證一個正數 x 嘅absolute value 少於 A，通常咁做
> 
> Prove x > 0 Prove x < A Conclude |x| <A
> 
> 負數嗰部分，我地唔會喺line 2畫蛇添足寫成 -x < 0 < A

其實呢句係講緊邊度？

GitHub 個 review 可以俾你指住個 file 嘅 line number 嚟俾 review comment 㗎
你去個 Files Changed tab 嗰度，就可以睇到啲 tex source，highlight 一行 source，左手邊就會出現一個藍色加號，就可以係嗰度 comment。

**Comment by ceciliachan1979** [comment]
> Prob 1
> 
> Prob 1.3: 問嘅係 p => q 係 true 定 false 。簡單答 false 即可。
> 
> Other parts are correct.

其實 problem 1.3 咁寫，主要係驚個 course instructor 淨係講 R，從來冇教過其它 metric space 同 metric space 唔一定 complete，in that case 答 false 就會錯，所以戴個頭盔啫 ⋯

**Comment by ceciliachan1979** [comment]
> Prob 7 idea 無錯，但presentation 可以更rigorous。例如第一段 "The Bolzano-Weierstrass theorem states that every bounded sequence has a convergent subsequence. The sequence {a_n } is bounded, so it has a convergent subsequence {a_{n_k} }." 我會改成 "The Bolzano-Weierstrass theorem states that every bounded sequence has a convergent subsequence; **in other words,** if a sequence {a_n } is bounded, then it has a convergent subsequence {a_{n_k} }."
> 
> suffix 係computer science term; 用返數學term, subsequence 。

suffix 同 subsequence 係唔同㗎⋯
suffix 係 `{a_7,a_8,a_9,...}`，後面啲 elements 我全部都要
subsequence 可以係 `{a_7, a_{10}, a_{23}, ...}`，後面啲 elements 可以選擇性唔要

**Comment by E7-87-83** [comment]
> suffix 同 subsequence 係唔同㗎⋯ suffix 係 `{a_7,a_8,a_9,...}`，後面啲 elements 我全部都要 subsequence 可以係 `{a_7, a_{10}, a_{23}, ...}`，後面啲 elements 可以選擇性唔要

OK. Apology.

**Comment by ceciliachan1979** [comment]
> > suffix 同 subsequence 係唔同㗎⋯ suffix 係 `{a_7,a_8,a_9,...}`，後面啲 elements 我全部都要 subsequence 可以係 `{a_7, a_{10}, a_{23}, ...}`，後面啲 elements 可以選擇性唔要
> 
> OK. Apology.

咁其實又唔駛道歉嘅﹐做錯數啫，我條條都有啲錯㗎啦，要道歉我咁多題做錯咪有排煩 😅 ⋯
尤其係其實係你啱，呢個位係應該用 subsequence

**Pull request merged**